src/sonic-yang-models/yang-models/sonic-bgp-device-global.yang

```diff
@@ -21,12 +21,26 @@ module sonic-bgp-device-global {
         container BGP_DEVICE_GLOBAL {
             container STATE {
                 description "BGP device-specific global data";
-                leaf tsa_enabled {
+                leaf tsa_enabled {                    
                     type boolean;
                     default "false";
                     description
                     "When set to true, Traffic is shifted away (TSA), i.e, BGP routes are not advertised to neighboring routers";
                 }
+                leaf idf_isolation_state {                    
+                    type enumeration {
+                        enum isolated_no_export {
+                            description "IDF isolated using no-export community tag";
```

Comment:
>IDF isolated using no-export community tag [](http://example.com/codeflow?start=41&length=42)

Can we add DC size - Medium/Large

Reply:
This field is just meant to indicate that the isolation is to be done by advertising routes tagged with no-export BGP community and is independent of any DC size.